src/controller/CHIPDeviceController.cpp

```diff
@@ -289,6 +289,18 @@ CHIP_ERROR DeviceController::GetPeerAddressAndPort(PeerId peerId, Inet::IPAddres
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DeviceController::GetPeerAddressInterfaceAndPort(PeerId peerId, Inet::IPAddress & addr, Inet::InterfaceId & iface,
```

Comment:
Please don't add new APIs using PeerId.  Yes, I know we have GetPeerAddressAndPort already....

Reply:
Pass in a ScopedNodeId or just a node id, depending on whether you want to assume that the DeviceController is bound to a fabric.  Then have this API impl here create the PeerId it needs for CASESessionManager, until we get that fixed.  That will reduce the scope of future changes as we fix CASESessionManager.

Reply:
Done! Also realized there was no need to pull apart the PeerAddress like the method above -- just returned the PeerAddress directly which is much simpler.